### PR TITLE
Add installation instructions with conda

### PR DESCRIPTION
**Pull request by martinRenou**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,13 @@ Spreadsheet in the Jupyter notebook:
    
 # Installation
 
-To install use pip:
+The easiest way to install ipysheet is with conda:
```
Comment (maartenbreddels) [inline README.md]: easiest? I'm not sure we should have an opinion about it, currently it's for sure the slowest :)

Comment (martinRenou) [reply README.md]: I can write the slowest then :P. I'll remove this comment, thanks for the review :)

**Comment by martinRenou** [comment]
Are you happy with this one?

**Comment by maartenbreddels** [comment]
yes 👍

**Pull request merged**